pep-0540.txt

```diff
@@ -270,27 +290,35 @@ Proposal
 Changes
 -------
 
-Add a new UTF-8 mode, disabled by default, to ignore the locale and
+Add a new UTF-8 mode, enabled by default in the POSIX locale, but otherwise
+disabled by default, to ignore the locale and
 force the usage of the UTF-8 encoding with the ``surrogateescape`` error
 handler, instead using the locale encoding (with ``strict`` or
 ``surrogateescape`` error handler depending on the case).
 
-Basically, the UTF-8 mode behaves as Python 2: it "just works" and don't
-bother users with encodings, but it can produce mojibake. It can be
-configured as strict to prevent mojibake: the UTF-8 encoding is used
-with the ``strict`` error handler for inputs and outputs, but the
-``surrogateescape`` error handler is still used for `operating system
-data`_.
+The "normal" UTF-8 mode uses ``surrogateescape`` on the standard input and
+output streams and openeded files, as well as on all operating system
```

Comment:
s/openeded/opened/
But what "opened files" means? Buitin `open`'s erros is None by default, and it equals to 'strict'.
Does this mean changing it?